x/arkeo/types/keys.go

```diff
@@ -3,7 +3,7 @@ package types
 const (
 	// ModuleName defines the module name
 	ModuleName   = "arkeo"
-	ReserveName  = "reserve"
+	ReserveName  = "arkeo-reserve"
```

Comment:
probably don't need to prepend `arkeo` here.

Reply:
Please see for the reason this is needed.  https://github.com/arkeonetwork/arkeo/pull/105